src/rpc/RPCHelpers.cpp

```diff
@@ -836,7 +836,9 @@ traverseOwnedNodes(
     for (auto i = 0; i < objects.size(); ++i)
     {
         ripple::SerialIter it{objects[i].data(), objects[i].size()};
-        atOwnedNode(ripple::SLE{it, keys[i]});
+        ripple::SLE sle(it, keys[i]);
+
+        atOwnedNode(sle);
```

Comment:
Does not look like you need this temporary but if you really want to have it you would have to `std::move()` it if you left the signature of above callback as it should be (`void(ripple::SLE&&)`).